extensions/http-curl/tests/HTTPHandlers.h

```diff
@@ -15,34 +15,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#ifndef LIBMINIFI_TEST_CURL_TESTS_SITETOSITEHTTP_HTTPHANDLERS_H_
+#define LIBMINIFI_TEST_CURL_TESTS_SITETOSITEHTTP_HTTPHANDLERS_H_
+
 #include "civetweb.h"
 #include "CivetServer.h"
 #include "concurrentqueue.h"
 #include "CivetStream.h"
 #include "io/CRCStream.h"
 #include "rapidjson/document.h"
+#include <cinttypes>
+#include <utility>
 
-#ifndef LIBMINIFI_TEST_CURL_TESTS_SITETOSITEHTTP_HTTPHANDLERS_H_
-#define LIBMINIFI_TEST_CURL_TESTS_SITETOSITEHTTP_HTTPHANDLERS_H_
 static std::atomic<int> transaction_id;
 static std::atomic<int> transaction_id_output;
 
 class FlowObj {
  public:
-  FlowObj()
-      : total_size(0) {
+  FlowObj() = default;
 
-  }
-  explicit FlowObj(const FlowObj &&other)
-      : total_size(std::move(other.total_size)),
+  FlowObj(FlowObj &&other) noexcept
+      : total_size(other.total_size),
         attributes(std::move(other.attributes)),
-        data(std::move(other.data)) {
+        data(std::move(other.data))
+  { }
 
-  }
-  uint64_t total_size;
+  uint64_t total_size{0};
```

Comment:
I know this is old code, but as we touch this, could you rename the members to follow the regular coding style we use? (ending with underscore)

Reply:
I think these are correctly not using a trailing underscore because they are public and I think struct naming applies to them. Would you be fine with me changing `FlowObj` to be defined with the `struct` keyword instead?

https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Okay, this makes sense, thanks!